openapi/components/schemas/Journal/JournalRecord.yaml

```
@@ -0,0 +1,91 @@
+type: object
+required:
+  - type
+  - journalEntryId
+  - customerId
+  - invoiceId
+  - invoiceItemId
+  - debitAccountId
+  - creditAccountId
+properties:
+  id:
+    $ref: ../ResourceId.yaml
+  journalEntryId:
+    description: ID of the journal entry.
+    readOnly: true
+    type: string
+  customerId:
+    description: ID of the customer.
+    type: string
+  invoiceId:
+    description: ID of the invoice.
+    type: string
+  invoiceItemId:
+    description: ID of the invoice item.
+    type: string
+  type:
+    type: string
+    readOnly: true
+    enum:
+      - automated
+      - manual
+  estimatedAmount:
+    description: |-
+      Amount of revenue estimated to be recognized at the schedule date.
+      Value is ignored when updating a journal record with `type` of `automated`.
```

suggestion: suggestion for readability 
```suggestion
      This value is ignored when updating a journal record with a `type` of `automated`.
```